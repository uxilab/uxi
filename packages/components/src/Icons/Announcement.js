import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Announcement = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 16 13.57"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <path d="M16,12.76V0H14.58V1L14.5,1,1.42,3.88V3.24H0V9.52H1.42V8.88l1,.23v2.94a.71.71,0,0,0,.54.69l3.24.81a.72.72,0,0,0,.74-.26l2-2.72,5.6,1.24v1Zm-9.9-.7L3.85,11.5V9.42l3.62.8Zm8.36-1.73-13-2.9V5.33l.08,0L14.58,2.4v8Z" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Announcement.propTypes = {
  color: PropTypes.string,
};

Announcement.defaultProps = {
  color: '#6d6d71',
};

export default Announcement;
