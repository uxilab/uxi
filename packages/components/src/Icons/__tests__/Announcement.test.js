import React from 'react';
import Announcement from '../Announcement';

describe('Icon : <Announcement />', () => {
  it('match snapshot', () => {
    expect(shallow(<Announcement />)).toMatchSnapshot();
  });
});

