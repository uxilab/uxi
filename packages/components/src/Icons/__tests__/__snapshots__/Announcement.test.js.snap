// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Announcement /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16 13.57"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M16,12.76V0H14.58V1L14.5,1,1.42,3.88V3.24H0V9.52H1.42V8.88l1,.23v2.94a.71.71,0,0,0,.54.69l3.24.81a.72.72,0,0,0,.74-.26l2-2.72,5.6,1.24v1Zm-9.9-.7L3.85,11.5V9.42l3.62.8Zm8.36-1.73-13-2.9V5.33l.08,0L14.58,2.4v8Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
